.github/workflows/semantic-release.yml

```diff
@@ -0,0 +1,38 @@
+name: Semantic Release
+
+on:
+  push:
+    # These are the branches that we expect to publish from
+    # semantic-release also has a regex that validates we're running on a release branch
+    branches:
+      - master
+      - "[0-9]+.x"
+      - "[0-9]+.x.x"
+      - "[0-9]+.[0-9]+.x"
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          persist-credentials: false
+      - uses: actions/setup-node@v1
+        with:
+          node-version: ">=14.16.0 14"
+      - run: npm ci
+      - name: Import GPG key
+        uses: crazy-max/ghaction-import-gpg@v3.0.1
+        with:
+          gpg-private-key: ${{ secrets.GPG_PRIVATE_KEY }}
+          passphrase: ${{ secrets.GPG_PASSPHRASE }}
+          git-user-signingkey: true
+          git-commit-gpgsign: true
+      - run: npx semantic-release
+        env:
+          # A personal access token is required to publish from protected branches
+          GITHUB_TOKEN: ${{ secrets.SEMANTIC_RELEASE_TOKEN }}
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+          GIT_COMMITTER_NAME: Sage Carbon Codemod
```

Comment:
I think this has to match the username on GitHub to link correctly which is `Sage Carbon`.